src/transformStateWithClones.js

```diff
@@ -8,6 +8,33 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const ADD_PROPERTIES = 'addProperties';
+  const REMOVE_PROPERTIES = 'removeProperties';
+  const CLEAR = 'clear';
+  const newState = { ...state };
+  const arrState = [];
+
+  for (const types of actions) {
+    if (types.type === ADD_PROPERTIES) {
```

Comment:
Use a destructuring for action here
```suggestion
  for (const action of actions) {
    if (types.type === ADD_PROPERTIES) {
```